prog-o-meter.py

```diff
@@ -108,6 +111,25 @@ def progress(self):
         """
         for i in range(self.days):        # Color a rectangle pr. completed day blue (from left to right)
             self.canvas.itemconfig(self.rectangle_list[i], fill = "blue") 
+    def get_completion_date(self,days_remaining):

```

Comment:
Yes you're right I would be able to acces it without of passing it as an argument. However in this case, when the user start the program, I wan't him to be presented with  completion_date - 1 day to take into account the current day. Then after the first click on "1 more day!", just the completion_date. 

Currently, the method returns the string directly so I can't simply do `self.completion_date = self.get_completion_date() - 1`

There are several way to get this done. The easiest and most flexible way I could think of at the time was to make days_remaining an argument. This way it's easy to do all sorts of calculation on the date when calling the method

Here are the other possibility I have:

* Make `get_completion_date` return a datetime object instead of a string, then do the calculation and the conversion to a string on the object it returns when called. I'm affraid this way will make the code harder to read.
* Add some sort of looping in the method to check for the first run. I prefer keeping the method simple so adding a loop is not ideal.

Let me know what you think. I'll wait for your input before I push.

Reply:
It makes a lot of sense, thank you for explaining it. I agree, keeping things simple and readable is great. If you add the space in line 114, and push, I'll merge 😀